setup.py

```diff
@@ -8,9 +8,10 @@
     'pytest-cov==2.4.0',
     'coveralls[yaml]==1.6.0',
     'pytest-xdist==1.18.1',
-    'py-evm==0.2.0a39',
-    'eth-tester==0.1.0b37',
-    'web3==5.0.0a6',
+    'py-evm==0.2.0a42',
+    'eth-tester==0.1.0b39',
+    'eth-abi==2.0.0b9',
```

Comment:
I don't think we need this dependency? (even for testing)

Reply:
Not pinning these burnt me in the past so, prefer to have to control now.